### PR TITLE
Update the gem

### PR DESCRIPTION
**Pull request by DeProdigy**
Raname the gem and update to Mini Racer

### CODE REVIEW

**Comment by noveng05** [comment]
<< domain

File: zxcvbn-js.gemspec [hunk 1]
```diff
@@ -13,10 +13,10 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.name          = "zxcvbn-js"
```
Comment (noveng05) [inline zxcvbn-js.gemspec:14]: hmm, does this need to change to `zxcvbnjs` too?

**Comment by noveng05** [comment]
tafn

File: zxcvbn-js.gemspec (hunk 1 above)
Comment (DeProdigy) [reply zxcvbn-js.gemspec:14]: We'll keep the gem name the same for now just to be compatible with the source. Maybe on day we will contribute back to them.`zxcvbnjs` is just namespacing the module so we can use 2 different ones at the same time.

**Comment by noveng05** [comment]
domainlgtm

**Comment by noveng05** [comment]
<< platformlgtm

**Pull request merged**